### ISSUE TITLE
Transfer to one of the Julia github orgs

### ISSUE THREAD

**Issue by ViralBShah** [simonster/Reexport.jl#43]
@simonster Would you be ok with transferring this package to one of the existing github orgs - so that others can help doing small maintenance?

It's a nice piece of work that actually doesn't require a whole lot!

**Comment by NeroBlackstone** [comment]
Agree, this package is so important that it should be used as some kind of infrastructure

**Comment by ViralBShah** [comment]
@simonster Would you be ok with transferring out this package?

**Comment by wheeheee** [comment]
Bump?

**Comment by ViralBShah** [comment]
I suspect he may not be seeing these notifications. Perhaps someone can email him at the email address listed on his github profile?

**Comment by simonster** [comment]
Sorry, my GitHub notifications get filtered out of my inbox and I usually ignore them but happened to randomly click on this one. I'm happy to transfer to another org if someone can tell me what org to send to!

**Comment by wheeheee** [comment]
I suppose ~~https://github.com/JuliaPackaging~~ the JuliaLang org itself? FunctionWrappers and Public.jl are there too

**Comment by ViralBShah** [comment]
@simonster Have temporarily bumped you to owner in JuliaLang to transfer there. Thanks!

cc @dilumaluthge

**Comment by DilumAluthge** [comment]
Yes, transferring to the JuliaLang org seems fine.

@simonster Let me know if you run into any problems when doing the transfer.

Sometimes GitHub removes people's admin permissions when a repo is transferred to an org. If that happens, ping me, and I'll restore your admin permissions on the repo.

**Comment by simonster** [comment]
Should be transferred!

**Comment by DilumAluthge** [comment]
Thanks! I've restored your admin permissions.

**Comment by ViralBShah** [comment]
Thanks @simonster
